### DISCUSSION TITLE
Error on Validation with sdk 1.7 ..

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#409] [Q&A] [answered]
Hi All !
I got sdk 1.7 but when I send the notice to Validation I got this error : 
2023-04-12 15:45:19,474 ERROR - 3.65.207.122 - /sdk/notice/en/sendToValidation.si - [ja-dev-as06.de.jaggaer.com#default task-24#00007] (com.bs.esop.labs.noticeforms.services.TedApiService jagCvsValidation 220)
[SC=BA232AUSR, IC=01000, VID=b8ed9e7a-c48c-49dc-9049-1e2d74939e08, AID=21862, UID=46104] - : org.springframework.web.client.HttpClientErrorException$BadRequest: 400 : "{"errorCode":400,"message":"The eForms SDK version 'eforms-sdk-1.7' does not exist.","timestamp":"2023-04-12T15:45:19Z"}"
Thanks a lot
Renato

**Comment by bertrand-lorentz** [comment, marked as answer]
Please note that SDK 1.7.0 has not been released yet: https://github.com/OP-TED/eForms-SDK/releases
So it is not available in our Validation service.

The SDK versions available in the Validation service on the "Preview" environment are indicated in: https://docs.ted.europa.eu/home/eforms/preview/index.html#_ted_central_validation_service

**Reply by IS-gmitov** [reply]
@bertrand-lorentz At the last eForms technical workshop it has been announced that SDK 1.7.0 will be released in April. At the moment the latest version on the link https://docs.ted.europa.eu/eforms/latest/index.html is 1.6.0. When SDK 1.7.0 will go official as latest version?
